.github/workflows/mirror-trivy-db.yaml

```diff
@@ -0,0 +1,22 @@
+name: Mirror Trivy DB
+run-name: ${{ github.actor }} is updating trivy DB mirror
+on:
+    workflow_dispatch:
+    schedule:
+        - cron: '0 */6 * * *'
```

Comment:
Every 6 hours feels too frequent. Every 1/2 days is better no?

Reply:
Trivy does it every 6 hours between ghcr and ecr